FoodsMenu/Menu.json

```diff
@@ -0,0 +1,16 @@
+{
+  "Foods": [
+    "Taart           - 5.99",
+    "Kaasplank       - 7.99",
+    "Snicker         - 0.99",
+    "Koek            - 0.75",
+    "Chips           - 1.25"
+  ],
+  "Beverages": [
```

Comment:
Bij de beverages klopt het formaat niet perfect omdat het nog niet geedite data is 😄